### ISSUE TITLE
[Bug] SP + i2v has quality issues

### ISSUE THREAD

**Issue by SolitaryThinker** [hao-ai-lab/FastVideo#583]
### Describe the bug

Definitely a problem with SP+i2v somewhere, but if you use learning rate of `1e5` or `2e5` the model will not break at least.

This i2v model can be finetuned without SP on bigger GPUs: https://huggingface.co/weizhou03/Wan2.1-Fun-1.3B-InP-Diffusers/tree/main

### Reproduction

Use SP > 1 and learning rate of 5e5 or higher with the following https://github.com/hao-ai-lab/FastVideo/blob/main/examples/training/finetune/wan_i2v_14b_480p/crush_smol/finetune_i2v.sh

### Environment

.

**Comment by Lyxien** [comment]
### Describe the bug

I hit the same bug when running `bash examples/training/finetune/wan_i2v_14b_480p/crush_smol/finetune_i2v.sh`，Starting around step 1300, the videos quickly begin to flicker and become blurry.

<img width="957" alt="Image" src="https://github.com/user-attachments/assets/5b9cc7f0-fc25-4682-a1cd-81ab8efe9410" />

<img width="909" alt="Image" src="https://github.com/user-attachments/assets/f7fd65f6-c277-4d61-acd2-41686ae7290d" />

### Reproduction

NUM_GPUS=4

training_args=(
  ……
  --num_latent_t 4
  ……
)

parallel_args=(
  --num_gpus $NUM_GPUS
  --sp_size $NUM_GPUS
  --tp_size $NUM_GPUS
  --hsdp_replicate_dim 1
  --hsdp_shard_dim $NUM_GPUS
)

I didn’t even add the --VSA-sparsity parameter.

**Comment by SolitaryThinker** [comment]
We are working on fixing this. SP+T2V works correctly and I2V without SP works and so far I still can't find the issue...

**Comment by Lyxien** [comment]
Could I kindly inquire whether a solution has been found for this issue?

**Comment by SolitaryThinker** [comment]
Not yet, I'm currently debugging this issue. Will update here with any new info

**Comment by SolitaryThinker** [comment]
should be fix in https://github.com/hao-ai-lab/FastVideo/pull/585. Do note that some arguments to finetune scripts have been updated
